src/common-utils/main.sh

```diff
@@ -455,20 +457,27 @@ if [ "${INSTALL_ZSH}" = "true" ]; then
             -c fetch.fsck.zeroPaddedFilemode=ignore \
             -c receive.fsck.zeroPaddedFilemode=ignore \
             "https://github.com/ohmyzsh/ohmyzsh" "${oh_my_install_dir}" 2>&1
-        echo -e "$(cat "${template_path}")\nDISABLE_AUTO_UPDATE=true\nDISABLE_UPDATE_PROMPT=true" > ${user_rc_file}
-        sed -i -e 's/ZSH_THEME=.*/ZSH_THEME="devcontainers"/g' ${user_rc_file}
 
         # Add Dev Containers theme
         mkdir -p ${oh_my_install_dir}/custom/themes
         cp -f "${FEATURE_DIR}/scripts/devcontainers.zsh-theme" "${oh_my_install_dir}/custom/themes/devcontainers.zsh-theme"
         ln -s "${oh_my_install_dir}/custom/themes/devcontainers.zsh-theme" "${oh_my_install_dir}/custom/themes/codespaces.zsh-theme"
 
+        # Attempt adding devcontainers .zshrc, does nothing given an existing file
+        if [[ (! -f "$user_rc_file") && ("$INSTALL_OH_MY_ZSH_CONFIG" = "true") ]]; then
```

Comment:
> (! -f "$user_rc_file") 

Question: Provided `installOhMyZshConfig:true`, shouldn't the contents be simply overwrriten by this Feature? If a user does not want that behavior, they can always pass `installOhMyZshConfig:false`

Reply:
We'd also need to update the description https://github.com/devcontainers/features/pull/614/files#diff-6dcca292e999f6e647eaaf8dec68633d7f1631c75140eeaf8b89683f0edb2bdcR26 (if we remove this condition)

Reply:
The reason it now does not override is to support base dockerfiles that have an existing .Ashcroft

Reply:
If that's the case, can the user pass `installOhMyZshConfig:false` to get around it?

My major concern with this change is that the Feature will start behaving differently than before (which might not be expected and could lead to unwanted changes). To make a change like this, we'd need to bump major version of this Feature. We avoid bumping major versions (unless necessary), as it's bit slow for everyone to update their dev containers, people will be left behind and lose important upcoming updates.

We can open an issue and add this change in when we are going to bump major version. @naturedamends what do you think?

Reply:
https://github.com/devcontainers/features/pull/614/commits/ab94a9760bb8962ee6113330d080e75308916198 Thanks! ❇️